console.lua

```diff
@@ -97,3 +101,107 @@ function BuffDuty:convertPlayerList(identifier, input)
     end
     return result
 end
+
+function Console.parseMessageCommand(cmd, ...)
+    local arg = {...} -- Arg list
+    local idx = 1 -- Current Arg index
+
+    local function arg_valid(i)
+        i = i or idx
+        return (arg[i] and not (i == #arg)) -- Not nil, and ignore the last Arg that AceConsole appends
+    end
+    
+    -- Print Usage Help
+    if arg[1] == "?" or arg[1] == "help" then
```

Comment:
I winder if it would make sense to add "-h" argument - very popular on unix systems.

Reply:
Doesn't hurt to have it. Done.